src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -1632,8 +1632,7 @@ private static PSObject GetParameterType(Type parameterType)
                 new ArrayList(Enum.GetValues(parameterType)) : new ArrayList();
             returnParameterType.Properties.Add(new PSNoteProperty("EnumValues", enumValues));
 
-            bool hasFlagAttribute = (isArray) ?
-                ((parameterType.GetCustomAttributes(typeof(FlagsAttribute), true)).Length > 0) : false;
+            bool hasFlagAttribute = (isArray) && ((parameterType.GetCustomAttributes(typeof(FlagsAttribute), true)).Length > 0);
```

Comment:
```suggestion
            bool hasFlagAttribute = isArray && ((parameterType.GetCustomAttributes(typeof(FlagsAttribute), true)).Length > 0);
```

Reply:
We can remove more redundant parenthesis in new PR.

Reply:
I mean we can use an automated code fix, rather than picking changes manually.